LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 1970 Nobody
```

Comment:
```suggestion
   Copyright NaN undefined
```

Diff suggestion of @eavichay's [suggestion](https://github.com/kelseyhightower/nocode/pull/4194/files#r499217063)